src/M/KarenJSON.hs

```diff
@@ -33,7 +33,7 @@ parseMenus' = withObject "day" $ \obj -> do
   name <- obj .: "name"
   recs <- obj .: "recipes"
   for recs $ withObject "recipe" $ \rec' -> do
-    [Object sv, Object eng] <- rec' .: "displayNames"
+    [Object sv, Object _] <- rec' .: "displayNames"
```

Comment:
The reason that I chose to name this object is that the api keeps the english version of the display name here. So if we want to add a way to change language it is sort of documented in the code where we need to switch it in the data.